queries/c_sharp/highlights.scm

```diff
@@ -157,7 +216,7 @@
   (identifier) @type)
 
 ; Generic Types.
-(type_of_expression
+(typeof_expression
   (generic_name
     (identifier) @type))
```

Comment:
Could the `generic_name` be generalized or no? E.g. I see right now for 
```csharp
        protected override Task<QuickFixResponse> TranslateResponse(QuickFixResponse response, GotoFileRequest request)
        {
            return response.TranslateAsync(Workspace, request);
        }
```
the `Task` part of `Task<QuickFixResponse>` is highlighted as a variable instead of a type

Reply:
no, consider this case - `MyFunction<A,B>(1);` where MyFunction is the identifier of a generic name. It is not a type but rather a function

Reply:
but yeah it's annoying to have to account for all these spots with generic name

Reply:
ah i see that makes sense!